plugins/chat/app/controllers/chat/api/channel_messages_controller.rb

```diff
@@ -34,7 +34,7 @@ def create
     Chat::MessageRateLimiter.run!(current_user)
 
     with_service(Chat::CreateMessage) do
-      on_success { render json: success_json.merge(message_id: result[:message].id) }
+      on_success { render json: success_json.merge(message_id: result[:message_instance].id) }
```

Comment:
The reason for this change is explained in services/chat/create_message.rb file changes